pkg/controllers/dynakube/logmonitoring/daemonset/env.go

```diff
@@ -17,8 +17,8 @@ const (
 	entityEnv        = "DT_ENTITY_KUBERNETES_CLUSTER"
 
 	// main container envs
-	apiNodeNameEnv  = "KUBELET_API_NODENAME"
-	apiIPAddressEnv = "KUBELET_API_ADDRESS"
+	ApiNodeNameEnv  = "KUBELET_API_NODENAME"
```

Comment:
```suggestion
	APInodeNameEnv  = "KUBELET_API_NODENAME"
```

Reply:
API is abbreviation.

Reply:
if I do that then I have to do `APIIPAddressEnv` which is horrible (or more horrible than `ApiIPAddressEnv` 😛 )

Reply:
nope, `APIipAddressEnv `

Reply:
that is not better, maybe even worse xd

Reply:
just call it `kubeletNodeName` then

Reply:
https://learn.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN

>When using acronyms, use Pascal case or camel case for acronyms more than two characters long. For example, use HtmlButton or htmlButton. However, you should capitalize acronyms that consist of only two characters, such as System.IO instead of System.Io.

I'm okey to rename it as @chrismuellner  suggested.

Reply:
that is a C# thing, at that point we could just use snake_case (which is the best IMO 😉 )

(but I will rename it)

Reply:
i know i know :) just posted as alternative opinion.

Reply:
[fixed](https://github.com/Dynatrace/dynatrace-operator/pull/4140/commits/2acd00c27cb0287b355e80d2bf478fcc9d5a8582)